scripts/utils/formatNumber.js

```diff
@@ -0,0 +1,19 @@
+export default (number, isPercentage)  => {
+  let parsedNumber;
+
+  if (number === null || number === 'undefined' || typeof number !== 'number') return 'N/A';
```

Comment:
Can number be a string representing a number sometimes ? (ie `'42'`). In that case maybe we should do a parseFloat, then check that `typeof === 'number'`